tests/qos/tunnel_qos_remap_base.py

```diff
@@ -74,7 +105,7 @@ def get_queue_counter(duthost, port, queue, clear_before_read=False):
     for line in output:
         fields = line.split()
         if fields[1] == txq:
-            return int(fields[2])
+            return int(fields[2].replace(',', ''))
 
```

Comment:
Why do we need this change?

Reply:
The "human readable" output for the CLI being invoked here can include comma characters, e.g. `1,000`. These need to be removed before passing to `int`.

Reply:
I see. Make sense.